roles/install/tasks/3_patch.yml

```diff
@@ -12,10 +12,10 @@
   shell: yum -q check-update > /dev/null
   register: patch_lvl
   failed_when: patch_lvl.rc not in [100,0]
-  become: yes
 
```

Comment:
Did you actually want to remove the `become: yes` or did you just want to make it `become: true`